test/WebJobs.Script.Tests/TestScripts/CSharp/AssembliesFromSharedLocation/run.csx

```diff
@@ -0,0 +1,9 @@
+﻿#r "..\SharedAssemblies\PrimaryDependency.dll"
```

Comment:
That was the idea. The implementation of the types contained in those two assemblies is so simple, I considered compiling/creating them as part of the tests. That might still be a good approach as it would avoid having things out of sync if we commit the code those assemblies "should" have. Thoughts?